### PR TITLE
Jinghan/refactor `sqlutil.Export`: adapt with bigquery store

### PR DESCRIPTION
**Pull request by jinghancc**
This PR does:
- modify `sqlutil.Export`, to make it adapt with bigquery offline store
- use `sqlutil.Export` for bigquery method `Export`

related to #802

### CODE REVIEW

**Review by YikSanChan — APPROVED** [review]
lgtm

File: internal/database/offline/sqlutil/export.go [hunk 1]
```diff
@@ -13,24 +13,50 @@ import (
...
-		defer close(errs)
 		errs <- errdefs.WithStack(err)
-		return stream, errs
+		return emptyStream, errs
 	}
 
+	return opt.QueryResults(ctx, dbOpt, opt.ExportOpt, query, args)
+}
+
+func sqlxQueryExportResults(ctx context.Context, dbOpt dbutil.DBOpt, opt offline.ExportOpt, query string, args []interface{}) (<-chan types.ExportRecord, <-chan error) {
```
Comment (lianxmfor) [inline internal/database/offline/sqlutil/export.go:52]: `sqlxQueryExportResults` supports bigquery, would it be more appropriate to remove the sqlx prefix?

Comment (jinghancc) [reply internal/database/offline/sqlutil/export.go:52]: no it doesn't support bigquery, bigquery has its own query function

Comment (lianxmfor) [reply internal/database/offline/sqlutil/export.go:52]: my bad.

**Review by lianxmfor — APPROVED** [review]
lgtm

**Pull request merged**